examples/volume-viewer-demo.js

```diff
@@ -98,6 +99,7 @@ $(function() {
         viewer.loadVolumes({
           volumes: [
             {
+              show_volume: false,
```

Comment:
I think you may be able to achive the same by not setting the `element_id` or `viewer_insert_class` or missing the whole `template: {...}` section... Worth trying at least...

Reply:
our current client implementation doesn't provide template: {...} at all and the volume viewer still show.